packages/desktop-client/src/components/common/Button.tsx

```diff
@@ -149,8 +149,9 @@ const Button = forwardRef<HTMLButtonElement, ButtonProps>(
         {...(typeof as === 'string'
           ? // eslint-disable-next-line @typescript-eslint/no-explicit-any
             (css(buttonStyle) as any)
-          : { style: buttonStyle, type: isSubmit ? 'submit' : 'button' })}
+          : { style: buttonStyle })}
         disabled={disabled}
+        type={isSubmit ? 'submit' : 'button'}
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏this change is unnecessary

Reply:
Never mind.. took me a while to understand this.

Reply:
A bit confusing since we have two kind of button types; the html button and our own